api/classes/publisher.js

```diff
@@ -1,411 +1,412 @@
-var Promise = require('bluebird'); // jshint ignore:line
-var mime = require('mime');
-var Path = require('path');
+"use strict";
 
-var client = require('../../lib/amazon-client').create();
+const Promise = require(`bluebird`);
+const mime = require(`mime`);
+const Path = require(`path`);
 
-var log = require('../../lib/logger.js');
-var Remix = require('../../lib/remix');
+const s3Client = require(`../../lib/s3-client`).connect();
 
-var Projects = require('../modules/projects/model');
-var PublishedProjects = require('../modules/publishedProjects/model');
+const log = require(`../../lib/logger.js`);
+const Remix = require(`../../lib/remix`);
 
-// SQL Query Generators
-var UserQueries = require('../modules/users/model').prototype.queries();
-var ProjectQueries = Projects.prototype.queries();
-var PublishedProjectQueries = PublishedProjects.prototype.queries();
-var PublishedFileQueries = require('../modules/publishedFiles/model').prototype.queries();
+const Projects = require(`../modules/projects/model`);
+const PublishedProjects = require(`../modules/publishedProjects/model`);
 
-var reject  = Promise.reject;
+// SQL Query Generators
+const usersQueryBuilder = require(`../modules/users/model`).prototype.queryBuilder();
```

Comment:
Because the exports for this require exposes something that is generated by an external library (viz. Bookshelf) which does not use ES6.